.gitignore

```diff
@@ -31,13 +31,12 @@ Session.vim
 
 # Config files
 sftp-config.json
-framework/test/network/pm2.network.json
-framework/test/network/configs/
-framework/test/network/networkTestsLogger.logs
+framework/test/mocha/network/pm2.network.json
+framework/test/mocha/network/configs/
+framework/test/mocha/network/networktest/mocha/sLogger.logs
```

Comment:
`sLogger.logs`? wired file naming. is it dynamically generated file? proper file name convention will be a good start.

Reply:
Ah good catch, thanks.